From 5004bf3dbf043b23aee993d1c7c1ac2fbd376cb2 Mon Sep 17 00:00:00 2001
From: "zongren.yin" <zongren.yin@amlogic.com>
Date: Fri, 3 Nov 2023 12:04:48 +0800
Subject: [PATCH 2/5] uboot: Add usb burn [1/1]

PD#SH-17103

Problem:
U-disk upgrade failed

Solution:
Add usb burn

Fixed conflicts:
NA

Verify:
local t7 build

Signed-off-by: zongren.yin <zongren.yin@amlogic.com>
Change-Id: I05a66ef0de9a445dffe615b7731954156b28811c
---
 board/amlogic/defconfigs/t7_an400_lpddr4x_debian_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board/amlogic/defconfigs/t7_an400_lpddr4x_debian_defconfig b/board/amlogic/defconfigs/t7_an400_lpddr4x_debian_defconfig
index 023e9e0e7e3..6b2d695da1a 100644
--- a/board/amlogic/defconfigs/t7_an400_lpddr4x_debian_defconfig
+++ b/board/amlogic/defconfigs/t7_an400_lpddr4x_debian_defconfig
@@ -93,6 +93,7 @@ CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_XHCI_CRG=y
 CONFIG_USB_GADGET=y
 CONFIG_USB_GADGET_CRG=y
+CONFIG_AML_FACTORY_BURN_LOCAL_UPGRADE=y
 CONFIG_AML_V3_FACTORY_BURN=y
 CONFIG_AML_V3_USB_TOOl=y
 CONFIG_WDT=y
-- 
2.25.1

